src/routes/virtual-scroll/+page.svelte

```
@@ -0,0 +1,579 @@
+                $pageRows
+                    .slice(0, 5)
+                    .map((r) => r.id)
+                    .join(',') + ($pageRows.length > 5 ? '...' : '')
+        })
+    })
+</script>
+
+<main>
+    <h1>Virtual Scroll Example</h1>
+    <p>
+        <a href="/">&larr; Back to examples</a>
```

ESLint

[error] Unexpected href link without resolve().

(svelte/no-navigation-without-resolve)